### PR TITLE
Fix/ecosystem-tab

### PR DESCRIPTION
**Pull request by Bluesmile82**
https://vizzuality.atlassian.net/browse/NCP-77?focusedCommentId=31707&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-31707

Changes on the comments
Also fix the scroll position on clicking the triggers and menu

### CODE REVIEW

**Review by clementprdhomme — COMMENTED** [review]
- On desktop, the “Dependencies” tab has horizontal overflow
- On mobile, the height limit of the tab content makes it less usable. I wonder if we should ditch any height limit. I think Teona meant that the tabs should be sticky at the top of the viewport, not that the tab content container should be scrollable.

**Comment by Bluesmile82** [comment]
I have been trying to make it work with the sticky behaviour but as the accordion has items inside and by default has overflow hidden is not possible to use sticky. Also the tabs have the items inside too. I could create the accordion component from scratch maybe. Can you think of another option?

**Comment by clementprdhomme** [comment]
In that case, I would just increase the min-height of the accordion up to desktop. Something maybe like 100vh or more.

**Comment by Bluesmile82** [comment]
Do you mean the accordion content? Its adjusted so we can see the title and tabs. Making it higher wont help

**Comment by clementprdhomme** [comment]
Oh ok, got it. I was testing with an iPhone 5 sized screen and it is barely usable but testing with more recent (small) phones, it's ok.

Then it's only the vertical overflow that we should look at. On mobile it's also present on the first tab. Anything we can do there? We'll have ugly scrollbars on Windows.

**Comment by Bluesmile82** [comment]
We have to have vertical overflow to make it work. I removed the scrollbars but Teona asked them back in. I'm thinking of redoing the accordion. Not sure if this is usable enough

File: src/app/industry-use-cases/page.tsx [hunk 1]
```diff
@@ -152,9 +173,13 @@ const IndustryUseCasesPage: FC = () => {
...
                   aria-expanded={activeItem === accordionItem.id}
-                  onClick={() =>
-                    setActiveItem(activeItem === accordionItem.id ? "" : accordionItem.id)
-                  }
+                  onClick={() => {
+                    setActiveItem(activeItem === accordionItem.id ? "" : accordionItem.id);
+                    const triggerRef = accordionItemsRef.current?.[index];
+                    setTimeout(() => {
+                      triggerRef?.scrollIntoView({ behavior: "smooth" });
+                    }, 150);
```
Comment (clementprdhomme) [inline src/app/industry-use-cases/page.tsx]: The accordions take 200ms to open/close, so this animation should be at least 200 or even 250ms just to make sure we scroll to the correct position.

**Review by clementprdhomme — APPROVED** [review]

**Pull request merged**